src/embed.js

```diff
@@ -59,7 +59,7 @@ function load(url, arg, prop, el, callback) {
 function embed(el, spec, opt, callback) {
   var cb = callback || function(){}, source,
   renderer = (opt && opt.renderer) || 'canvas',
-  actions  = (opt && opt.actions) || {}, mode;
```

Comment:
I guess the default was no actions at all, right? 

Also, this code is not correct. If `opt` is undefined, you call `opt.actions`